packages/sync/src/conflicts/ConflictResolutionStrategy.ts

```diff
@@ -19,6 +19,6 @@ export type ConflictResolutionStrategy =
  * @param client - client data
  */
 export interface ConflictResolutionStrategies {
- [operationName: string]: (server: ConflictResolutionData,
-                           client: ConflictResolutionData) => ConflictResolutionData;
+ [operationName: string]: (server: ConflictResolutionData, client: ConflictResolutionData) => ConflictResolutionData;
+  default: (server: ConflictResolutionData, client: ConflictResolutionData) => ConflictResolutionData;
```

Comment:
I'm not sure but it feels that this interface will overlap itself. 
[operationName: string] will match everything including "default" key. 
I would add a separate key for this just to avoid problems.